app/javascript/components/HelloReactRails.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from "react"
+// import PropTypes from "prop-types"
```

Comment:
- [OPTIONAL] Please kindly remove commented code from your code base, as it makes your code base dirty and less readable.